### ISSUE TITLE
Issues with DrCIF

### ISSUE THREAD

**Issue by TonyBagnall** [uea-machine-learning/estimator-evaluation#10]
EMOPain crashes with the following error
![image](https://user-images.githubusercontent.com/9594042/193555140-ba9abb2f-4af2-4b79-bc24-ae456d7baa48.png)

**Comment by MatthewMiddlehurst** [comment]
The image seems to be running TSFreshClassifier instead of DrCIF. One issue with DrCIF is that it is rather slow currently, I believe about 80% of the time spent is in the Catch22 transformer.

**Comment by MatthewMiddlehurst** [comment]
from #2
![](https://user-images.githubusercontent.com/9594042/190360521-02317759-a0d1-490b-9739-4390981d5c7e.png)

**Comment by TonyBagnall** [comment]
yeah I will sort through all this next week, wonder how long TDE will take on FaceDetection, currently at 350 hrs
 :)
